components/post-body.tsx

```diff
@@ -52,6 +56,11 @@ export default function PostBody({ content, authorName }) {
         '<div id="author-description-placeholder"></div>' // Placeholder for AuthorDescription
       );
 
+    replacedContentWithAuthorDescription =
+      replacedContentWithAuthorDescription.replace(
```

Comment:
can you please explain this part why you have done this?

Reply:
It's captures the class for removing default author part in the blog which comes with wordpress data.